src/js/util.js

```diff
@@ -6,12 +6,35 @@ var Wand = (function(wand) {
   wand = wand || {};
   wand.util = {};
 
-  wand.util.loadXhr = function(url, callback) {
-    console.debug('Retrieving url ' + url);
-    return getRequest(url, callback);
+  wand.util.encodeParams = function(objParams) {
+    var str = [];
+    for(var p in objParams) {
+      if (objParams.hasOwnProperty(p)) {
+        str.push(encodeURIComponent(p) + "=" + encodeURIComponent(objParams[p]));
+      }
+    }
+    return str.join("&");
   };
 
-  function getRequest(url, callback) {
+  wand.util.loadXhr = function(trigger, params, callback) {
+    console.debug('Retrieving url ' + trigger.url);
+    return getRequest(trigger.url, params, callback);
+  };
+
+  wand.util.loadJsonp = function(url, params, callback) {
+    var callbackName = 'jsonp_callback_' + Math.round(100000 * Math.random());
```

Comment:
@mapsam: So some APIs are standard APIs. Miami has to work with some APIs where there are CORS issues, so we've been using JSONP to get around that. The thought is we could add an existing `jsonp` attribute and the `loadJsonp` method would be used instead:

```
    "triggers": [
      {
        "content": "submit",
        "api": "http://still-eyrie-4551.herokuapp.com/areas",
        "preprocessor": "nested.preprocessor",
        "callbackFn": "nested.handleYesNoApi",
        "jsonp": true
      }
    ]
```